dot/parachain/network_protocols.go

```diff
@@ -12,7 +12,6 @@ import (
 
 const (
 	LEGACY_VALIDATION_PROTOCOL_V1 = "/polkadot/validation/1"
```

Comment:
just a nit for code consistency
```suggestion
	LegacyValidationProtocolV1 = "/polkadot/validation/1"
```